packages/discovery-provider/plugins/pedalboard/apps/relay/src/utils.ts

```diff
@@ -22,3 +21,24 @@ export const isUserDeactivate = (
 export const unknownToError = (e: unknown): Error => {
   return e instanceof Error ? e : new Error(String(e))
 }
+
+export const retryPromise = async <T>(task: () => Promise<T>, retries = 128): Promise<T> => {
```

Comment:
128 seems...a bit high? 😅 
Especially if the timeout between is half a second. This is going to probably exceed the HTTP timeout from most clients.
I think 3 times is the standard for most of the client code.

Reply:
we use 128 for the confirmations as well but yeah, i can lower the intensity a bit

Reply:
we do want it longer than the container restarts which are something like 30s iirc

Reply:
Ah I see. Carry on, then!